target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -169,6 +169,9 @@ ramips_board_detect() {
 	*"DWR-512 B")
 		name="dwr-512-b"
 		;;
+	*"DWR-922 E2")
```

Comment:
You can drop this as you use board name from DTS.

Reply:
Thanks. I hadn't originally made that change locally, but thought it might have been supposed to be there since the similar model was. I reverted the file and repushed it, or do we just remove the change from the pull request?